swagger-ui/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2018 Hammock and its contributors
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+  ~ implied.
+  ~
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>container-aware</artifactId>
+        <groupId>ws.ament.hammock</groupId>
+        <version>2.2-SNAPSHOT</version>
+        <relativePath>../container-aware</relativePath>
+    </parent>
+    <artifactId>swagger-ui</artifactId>
+    <dependencies>
+        <dependency>
```

Comment:
Are you expecting a user overrides this?  I see there's a config property for it, but a user also has to include the transitive dependency?

Reply:
Had to have that property anyway for syncing the packaged version to runtime info.
This because all webjars have there version in the resource path, so multiple versions don't conflict.
Have now added scanning bean so artifact ui version can be updated without extra steps.